Hi @Skyllarr, the build is currently failing.

Overall build process:
1. wildfly-elytron - SUCCESS 
2. wildfly-core (replace wildfly-elytron version with the one resulting from 1.) - SUCCESS
3. wildfly (replace wildfly-core version with the one resulting from 2.) - FAILURE

Details:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0-jboss-2:compile (default-compile) on project wildfly-undertow: Compilation failure
[ERROR] /home/fburzigo/projects/git/Skyllarr/wildfly/undertow/src/main/java/org/wildfly/extension/undertow/handlers/ReverseProxyHandlerHost.java:[169,19] cannot reference this before supertype constructor has been called
```

If it can be of any help, I could remove the issue by rebasing your wildfly `regex-role-mapper` branch against the upstream `master` and then the build was successful.

--- a/docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc
+++ b/docs/src/main/asciidoc/_elytron/Elytron_Subsystem.adoc
@@ -325,6 +325,10 @@ performs a logical operation using two referenced role mappers.
 
 |mapped-role-mapper |A role mapper definition for a role mapper that uses
 configured mapping of role names to map role names.
+
+|regex-role-mapper |A role mapper definition for a role mapper that
+performs a regex matching and maps matching roles with provided pattern.
+Regex can capture groups that replacement pattern can make use of.
 |=======================================================================
 
 [[ssl-components]]
@@ -665,6 +669,7 @@ and openssl provider loaders.
             }
         },
         "provider-sasl-server-factory" => {"global" => {"providers" => undefined}},
+        "regex-role-mapper" => undefined,
         "regex-principal-transformer" => undefined,
         "regex-validating-principal-transformer" => undefined,
         "sasl-authentication-factory" => {

